decentraland/unity-renderer
# style: clean up textures

This PR was created to: 
- Delete unused
- Delete duplicated assets with different names
- Move asset to the right folder
- Update assets configuration to make them more performant

In a nutshell, the **number of assets decreased from 373 to 173**. Also, needed textures would be easier to find.

Note that after the clean up many prefabs got broken because they were using the wrong texture, that is why there are many prefabs edited.

**How to test**
1- Open this [link](https://play.decentraland.org/?explorer-branch=style/CleanUpOldTextures)
2- Log in as a guest and complete all the steps of the tutorial
3- Walk through all the UI as a guest to check nothing is broken
4- Then go an connect you wallet
5- No complete the review of all the features that as guest are unavailable
